google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITTracingTest.java

```diff
@@ -59,7 +59,10 @@ public class ITTracingTest {
       Logger.getLogger(com.google.cloud.firestore.it.ITBaseTest.class.getName());
 
   private static final String SERVICE = "google.firestore.v1.Firestore/";
+  private static final String BATCH_GET_DOCUMENTS_RPC_NAME = "BatchGetDocuments";
```

Comment:
is this used in the actual code as well? May make sense to make it a protected variable so you don't have to copy over here (same goes for others)

Reply:
These tests ensure that 2 kinds of trace spans are generated:
1- spans generated in the Firestore handwritten layer.
2- spans generated by the gRPC auto-instrumentation layer.

For #1: we do reuse variables from our `TraceUtil.java` code.
For #2: the spans correspond to the gRPC request created in by the GAPIC layer, and there doesn't seem to be a variable defined that we can reuse here. See: https://github.com/googleapis/java-firestore/blob/532e094b1a428af2f03fd91202e718a428366b5a/grpc-google-cloud-firestore-v1/src/main/java/com/google/firestore/v1/FirestoreGrpc.java#L238